itools-packager/src/main/resources/itools.bat

```diff
@@ -50,13 +50,13 @@ if "%1"=="" ( goto done ) else (
 
 set options=
 if not "%itools_cache_dir%" == "" ( set options=-Ditools.cache.dir="%itools_cache_dir%" )
-if "%itools_config_dir%" == "" ( set itools_config_dir=%HOMEDRIVE%%HOMEPATH%\.itools)
-set options=%options% -Ditools.config.dir="%itools_config_dir%"
-if not "%itools_config_name%" == "" ( set options=%options% -Ditools.config.name=%itools_config_name%)
+if "%powsybl_config_dirs%" == "" ( set powsybl_config_dirs=%HOMEDRIVE%%HOMEPATH%\.itools)
```

Comment:
The default value of powsybl_config_dirs should be `%installBinDir%\etc:%HOMEDRIVE%%HOMEPATH%\.itools`

Reply:
(I think it is a typo so I put "%installDir%\etc:%HOMEDRIVE%%HOMEPATH%\.itools")